body.tex

```diff
@@ -59,11 +59,54 @@ \subsection{Basic Real/Bogus Classifier}
   \item output: reliability score for the potential transient
 \end{itemize}
 
-Although the output of a classifier is categorical (i.e. a class label), we will be using the output before the last Softmax~\footnote{TBD} layer, which can be intepretted as a probability.
 
-\subsubsection{Drawbacks}
+Regardless of how the score is calculated, the placement of the module in the pipeline makes it a score conditioned on the results of the upstream modules:
+\begin{equation}
+  \label{eq:conditional}
+  S_t =  Score(real | det)
+\end{equation}
+
+where \emph{det} is a flag representing whether a source is detected by the upstream modules or not, and is usually implemented as a $5\sigma$ thresholding of the output of the image differencing module -- see \cref{fig:diagram}. Note that since typical neural networks do not estimate probability distributions, we do not use a probabilistic notation in \cref{eq:conditional}.
+
+% Although the output of a classifier is categorical (i.e. a class label), we will be using the output before the last Softmax~\footnote{TBD} layer, which can be interpreted as a probability or score.
+
+\subsection{The score}
+As the task definition embraces a recognition according to spatial \emph{and} temporal features simultaneously, the score needs to be defined carefully to capture both aspects.
+More specifically, we need to define the term ``real'' more precisely: a non-astrophysical detection, such as an artifact, is obviously not ``real''. But how about a non-varying astrophysical source?
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=.8\textwidth]{material/score-space}
+  \caption{The 2-dimensional realness space}
+  \label{fig:score-space}
+\end{figure}
+
+As illustrated in figure~\ref{fig:score-space}, the real-ness space is composed of (at least) two dimensions, which we name ``transient-ness'' and ``object-ness''.
```

Comment:
While I understand what you mean here, I suspect if we could come up with other names it would be less confusing--"transient" and "object" already having technical definitions in LSST...

Reply:
I'm gonna need some help here, as I can't seem to find much better alternatives.
As for the term "transient", I think here it refers to the same concept as in LSST in general. Maybe something like "variability"? (but then again the conflict gets deeper!)

I think the term "object" is also too popular to be booked for a very narrow definition. Here I am trying to refer to astrophysical objects as opposed to artifacts. I can think of "real-ness", but we have already used that for something else :)

Any suggestions are appreciated :)

Reply:
"transient-ness" I think is a roundabout way of saying "signal to noise ratio"

"object-ness" is effectively 1-spuriousness, which we already use for this purpose in LSST.